components/Navbar.tsx

```diff
@@ -92,6 +93,12 @@ const Navbar = () => {
               />
             </>
           )}
+          <a
```

Comment:
Heller bruke `<Link>`?

Reply:
I Next.js burde man bruke Link komponenten for intern navigasjon på siden, ettersom den forehåndslaster inn siden for den blir trykket på, som gjør at routingen blir raskere. For eksterne sider, som til google forms er det bedre å bruke a-taggen, da den gjør at siden refresher helt når en ny side blir lasted inn.